command/report.go

```diff
@@ -52,7 +52,7 @@ Options:
 
   Commit Limiting:
 
-  -n int=1                   Limit output, 0 is no limits, defaults to 1 when no limiting flags otherwise defaults to 0
+  -n int=1                   Limit output, 0 is no limit, defaults to 0
```

Comment:
Change `-n int=1` -> `-n int=0`, as it is new default

Reply:
Oh, sorry, I completely oversaw that haha.